packages/clarity-js/types/core.d.ts

```diff
@@ -32,15 +30,11 @@ export const enum Setting {
     LongTask = 30, // 30ms
 }
 
-export const enum RegionFilter {
-    Url = 0,
-    Javascript = 1
-}
-
-export const enum Extract {
-    Text = 0,
-    Javascript = 1,
-    Attribute = 2
+export const enum Source {
```

Comment:
Source is too vague and broad a a name. Can you use ExtractSource or something narrower to clarify the meaning?